libnavigator/src/main/java/com/mapbox/navigation/navigator/internal/MapboxNativeNavigatorImpl.kt

```diff
@@ -48,21 +50,18 @@ import java.util.concurrent.CopyOnWriteArraySet
 import java.util.concurrent.Executors
 import kotlin.coroutines.resume
 
+private val TAG = Tag("MbxNativeNavigatorImpl")
```

Comment:
let's move `TAG` under `MapboxNativeNavigatorImpl ` object. When we keep it outer, in java bytecode creates an additional outer class that wraps `TAG`